sdk/core/System.Net.ClientModel/src/System.Net.ClientModel.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Contains building blocks for clients that call cloud services.</Description>
+    <Version>1.0.0-beta.1</Version>
+    <Nullable>enable</Nullable>
+
+    <TargetFrameworks>netstandard2.0;net6.0</TargetFrameworks>
```

Comment:
We should close with @KrzysztofCwalina that these are the desired TFMs before shipping this library GA.

Reply:
I like NET6 and NS2 as the TFMs